go.work

```diff
@@ -0,0 +1,7 @@
+go 1.21.5
+
+use (
```

Comment:
we should avoid committing this file, recommendation from golang, the pr i opened has the solution for it. We push a go.work.example and people can use that if they want.